docs/blog/20230501-Daeraxa-MayUpdate.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Community Update
+author: Daeraxa
+date: 2023-05-01
+category:
+  - news
+  - log
+tag:
+  - update
+---
+
+Is it a bird? Is it a plane? No it's the May community update!
+
+<!-- more -->
+
+# Welcome to the May Community Update
+
+Hello again and a warm welcome to our May community update! In store for you this time are Pulsar iconography updates, `symbols-view` improvements, CI upgrades and bunch of upgrades to our package backend and website.
+
+Just want to first mention that we have a [new survey](https://docs.google.com/forms/d/e/1FAIpQLSfnEEAQHUJbRVcH2NlNvLxmK0ZUcb5xuOCt1pYOGytvne0kJw/viewform?usp=sf_link) up, we would like some feedback from our community on a few aspects of the project so if you haven't answered this already then please have a look, every response is useful to us and we greatly appreciated.
+
+Now without further waffling, onto the updates!
+
+## Octicons iconography update
+
+So this is a change we [have in the works](https://github.com/pulsar-edit/pulsar/pull/509) thanks to [@confused-techie], to upgrade Pulsar's iconography by updating the version of [Octicons](https://primer.style/octicons/). The version being used currently in Pulasr is _really_ old (v4.4.0 - from 2016!) and there have been many, many updates since then with a whole bunch of new icons and improvements (now on version v18.3.0).
```

Comment:
Typo here on Pulsar (pulasr)

Reply:
Good catch, annoyingly I did see this myself but apparently never committed it...